tests/src/Common/hostpolicymock/HostpolicyMock.cpp

```diff
@@ -6,6 +6,7 @@
 // Used for testing CoreCLR/Corlib functionality which calls into hostpolicy.
 
 #include <string>
+#include "platformdefines.h"
```

Comment:
should be `<platformdefines.h>`?

Reply:
My personal preference is to only do `<>` for system headers, but I'll go change it.